.github/workflows/renovate.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
         uses: actions/checkout@v4
       - name: Setup node
         uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
+      - name: 'Configure Git username/email'
+        run: |
+          git config --global user.name "${GITHUB_ACTOR}"
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
```

Comment:
Is it working in this case? You have set in the previous PR `${{ github.actor_id }}` instead of `${GITHUB_ACTOR_ID}` and `${{ github.actor }}` instead of `${GITHUB_ACTOR}`

Reply:
Yes, the context is different.

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

Reply:
this code is run by the shell so it has the env vars available.